apis/apiextensions/v1/composition_transforms.go

```diff
@@ -18,15 +18,24 @@ package v1
 
 import (
 	"encoding/json"
+	"strconv"
 
 	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+
+	xperrors "github.com/crossplane/crossplane/pkg/validation/errors"
 )
 
 // TransformType is type of the transform function to be chosen.
 type TransformType string
 
 // Accepted TransformTypes.
 const (
+	ErrConvertFormatPairNotSupported = "conversion from %s to %s is not supported with format %s"
```

Comment:
nit: we use `errFmt` prefix as a convention to indicate the error message is a formatted string. https://github.com/crossplane/crossplane/blob/894e2324d8e2fe07688e52074098d889b78e90af/internal/controller/apiextensions/composite/composition_ptf.go#L73

```suggestion
	ErrFmtConvertFormatPairNotSupported = "conversion from %s to %s is not supported with format %s"
```